### ISSUE TITLE
PHM Pegasus (1988)

### ISSUE THREAD

**Issue by Burrito78** [TotalDOSCollection/TDCbugs#16]
File:
PHM Pegasus [a1] (1988)(Electronic Arts, Inc.) [Action, Simulation].zip

Issue:
DREAMM 3.0b15: Unfamiliar set of matching files
Note: CP is missing
Comment Aaron: HF.EXE has 9 extra 0 bytes at end; could be legit, so I added a patch in 3.0b16+ to patch it back to the original

 ![Bildschirmfoto 2024-06-25 um 11 34 50](https://github.com/TotalDOSCollection/TDCbugs/assets/61995128/6d78bc45-8d02-44fe-9131-1d6bbfa9fc48)

**Comment by TotalDOSCollection** [comment]
Every copy of this game I've located, going back as far as 1998 and every disk image I can locate show the size of HF.EXE as being 69,296 bytes, except [a1] which is sourced from Italsoft at 69,287 bytes.  Italsoft commonly hacked and otherwise modified files, so this copy should be ignored.  See PHMINST.BAT for a prime example in this same archive.

[a1] will go away from TDC and into the junk pile for tracking.

Related:
Added: PHM Pegasus (1988)(Electronic Arts, Inc.) [IMA][360k].zip

this is the only floppy image that looked legitimate to me.  Others clearly were copies of this one and/or constructed and showed obvious signs of being created within the last 2 decades.  This one is still modified with a mostly useless PHMFIX.EXE which oddly just launches the game and does not patch it to bypass the copy protection.

**Comment by TotalDOSCollection** [comment]
forgot to mention, I'd suggest removing the patch that accepts the 9 bytes less version of the file to be allowed.

**Comment by TotalDOSCollection** [comment]
I take back my comment about removing the 9 byte patch.  
After some sleepless 3am ideas, I decided to check out the exe header when I got a chance and it seems to be valid even on that italsoft version.  I'd somehow skipped over that byte 2 is different by exactly 9 bytes, showing that the size for that italsoft release is correct, as is the one we're currently using in TDC.  Yes it's possible that italsoft for some unknown reason resized the exe header or it was somehow re-written perhaps during an application of pklite and then removing it, but it does appear to be correct in its current form.  We clearly need more original floppy dumps on this title to really know the history.
